agents/fake_data/fake_data_agent.py

```diff
@@ -44,6 +44,28 @@ def start_acq(self, session, params=None):
 
         This Process has no useful parameters.
 
+        The most recent fake values are stored in the session.data object in
+        the format::
+
+            {"fields":
+                {"channel_00": 0.10250430068515494,
+                 "channel_01": 0.08550903376216404,
+                 "channel_02": 0.10481891991693446,
+                 "channel_03": 0.1060597011760155,
+                 "channel_04": 0.1019265554541543,
+                 "channel_05": 0.09389479275963578,
+                 "channel_06": 0.10071855402986646,
+                 "channel_07": 0.09601271802732826,
+                 "channel_08": 0.09760831143883832,
+                 "channel_09": 0.11345360178932645,
+                 "channel_10": 0.10047676575328081,
+                 "channel_11": 0.09534462609141414,
+                 "channel_12": 0.09654199950059912,
+                 "channel_13": 0.11051763608358373,
```

Comment:
It would probably be safe to leave out channels 02-13 here, and represent them with "..."...

Reply:
Since number of channels is sort of arbitrary I just trimmed it down to 4.